website/source/security.html.erb

```diff
@@ -0,0 +1,32 @@
+---
+layout: "inner"
+page_title: "Security"
+description: |-
+  Terraform takes security very seriously. Please responsibly disclose any security vulnerabilities found and we'll handle it quickly.
+---
+
+<h1>Terraform Security</h1>
+
+<p>
+  We understand that many users place a high level of trust in HashiCorp
+  and the tools we build. We apply best practices and focus on security to
+  make sure we can maintain the trust of the community.
```

Comment:
Hey James - that's good feedback. This is a general page we are putting across all our projects, but I'll take these bullet points and send them to our team to add to https://www.hashicorp.com/security.html, which is linked below.